src/preferences/broadcastprofile.cpp

```diff
@@ -76,6 +76,8 @@ const bool kDefaultNoDelayFirstReconnect = true;
 const bool kDefaultOggDynamicupdate = false;
 double kDefaultReconnectFirstDelay = 0.0;
 double kDefaultReconnectPeriod = 5.0;
+const QString kDefaultStreamName =
+        QObject::tr("Mixxx");
```

Comment:
```suggestion
const QString kDefaultStreamName = QStringLiteral("Mixxx");
```
As @Holzhaus said: Mixxx is not going to be translated, you can use QStringLiteral instead.

Reply:
I see so that's what the QStringLiteral meant!